articles/cms/wordpress/installation.md

```diff
@@ -49,28 +56,34 @@ Each of the options above has trade-offs. Option 1 has the least number of diffe
 
 As always, if you have any questions about this setup process, create a post on our [Community](https://community.auth0.com/tags/wordpress) tagged "wordpress."
 
-## Setup Wizard - Standard
-
-Click **Standard** to start the basic Auth0 setup to allow username + password, social, and Passwordless logins.
+## Setup Wizard
 
-![WordPress Installation select setup type](/media/articles/cms/wordpress/setup-wizard-select-setup.png)
+The Setup Wizard will attempt to create all the necessary components needed to use Auth0 on your WordPress site. If you have an existing Application or database Connection you want to use, please see the [Manual Setup](#manual-setup) steps below.
 
 ### Option 1: Standard Setup
 
-This will create and configure an Application and a database connection for this site.
+This will create and configure an Application and a database Connection for this site.
 
-![WordPress Installation standard setup fields](/media/articles/cms/wordpress/setup-wizard-social-modal.png)
+First, [follow the instructions here](/api/management/v2/get-access-tokens-for-test#get-access-tokens-manually) to generate an Auth0 Management API token. Once the token is generated, make a note of the domain name used in the "Identifier" field under the **Settings** tab. For example, if your Identifier is `https://tenant-name.auth0.com/api/v2/`, then the tenant Domain is `tenant-name.auth0.com`. [More about tenant Domains can be found here](https://auth0.com/docs/getting-started/the-basics#domains).
```

Comment:
@joshcanhelp The domain is shown in the same page where the token is copied under the "Using this token in the API Explorer" section, in step 3. That's what I felt was important to mention on the instructions to make it easier and smoother to get the domain name. What do you think?

![image](https://user-images.githubusercontent.com/37590801/59303278-dbdf3480-8c63-11e9-836a-d413db750445.png)